File: app/src/main/java/android/support/v4/app/SupportV4App.java
package android.support.v4.app;

/**
 * We need this to access some internal properties of the support library.
 * Good thing, we ship out a copy of the library with the app. If we didnt do that,
 * this might totally fail.
 */
public class SupportV4App {
    public static FragmentManager fragmentChildFragmentManager(Fragment fragment) {
        return fragment.mChildFragmentManager;
    }
}

File: app/src/main/java/android/support/v4/app/SupportV4App.kt
package android.support.v4.app

/**
 * We need this to access some internal properties of the support library.
 * Good thing, we ship out a copy of the library with the app. If we didnt do that,
 * this might totally fail.
 */
object SupportV4App {
    @JvmStatic
    fun fragmentChildFragmentManager(fragment: Fragment): FragmentManager {
        return fragment.mChildFragmentManager
    }
}